Replicator/Replicator.cc

```diff
@@ -311,45 +299,48 @@ namespace litecore { namespace repl {
 
     string Replicator::statusVString() const {
         std::stringstream ss;
-        for (CollectionIndex i = 0; i < _pushStatusV.size(); ++i) {
+        for (CollectionIndex i = 0; i < _subRepls.size(); ++i) {
```

Comment:
Please be aware that if this function is also used by the passive replicator, it may not work as expected.

Reply:
Yes, thanks for pointing it out. I may try to unify them when I come to the passive basics. Make an index mapping from, active collections to configured collection. For Active replicator, it would be i -> i. For Passive replicator, it will be non trivial mapping.